Kalela/BrightEvents
# Bg-login-error-returns-156599195

### What does this PR do?
Removes a bug in the login endpoint where the data returned is not json. It is good practice to return json data when designing an API.
### How should this be manually tested?
- Clone the repo locally.
- cd into the project.
- If pip is installed, run ```pip install -r requirements.txt```, otherwise, [install pip first](https://pip.pypa.io/en/stable/installing/).
- Run ```flask run``` in your terminal to get the server up.
- Register and Login using postman and check if the response from the API is json data.
### What are the relevant pivotal tracker stories?
Login error returns
ID: #156599195
